db/static_seeds.rb

```diff
@@ -8,3 +8,6 @@
   {start_date: "2017-05-28", name: "Bio", video: nil, duration: nil},
   {start_date: "2017-05-28", name: "Konventionell", video: nil, duration: nil}
 ])
+Sensor.create!([
```

Comment:
Maybe it's wrong to put it in the static_seeds. My intention is that I need to have all the sensors for all the cows present (created before) when I query the Smaxtec API and write the Sensor Readings for the associated Sensor.
Is the static_seeds file only the place for the seeds you expect in the test?